### DISCUSSION TITLE
MIB Browsing

### DISCUSSION THREAD

**Discussion by rupadana** [Cepat-Kilat-Teknologi/go-snmp-olt-zte-c320#11] [Q&A] [answered]
Halo, can you please share how u get the OID from the OLT?

at mine, i want to increase PON ID into 16, cause this project only support 1-8 pon ID

**Comment by sumitroajiprabowo** [comment, marked as answer]
at the moment i just have GTGO 8 port. for the next time i will update this repo if i use GTGH 16 port.
